content/blog/advanced-typescript-type-ftw/index.md

```diff
@@ -1,17 +1,19 @@
 ---
 title: "Advanced TypeScript type FTW!"
 authors: ["cyrus-najmabadi"]
-tags: ["TypeScript"]
+tags: ["Infrastructure as Code"]
```

Comment:
Other tags that have multiple words use a hyphen between the words (e.g. "Terraform-Migration"), whereas here it is using spaces. They all become lowercase with hyphens when rendered, so it doesn't really matter, but just wondering what the "naming guide" is for tags that's consistent and will make sense to the rest of the team.

Reply:
And indeed I came across another post that's using "Infrastructure-as-Code" for the tag.

Reply:
Good catch, I'll fix it. I have a Markdown file with various formatting suggestions and conventions, I was planning on getting that reviewed before sharing and/or checking in. But yeah, we need to be consistent for our tags.